commands/address_daemon_test.go

```diff
@@ -115,3 +117,26 @@ func TestWalletLoadFromFile(t *testing.T) {
 	wb := d.RunSuccess("wallet", "balance", fixtures.TestAddresses[0]).ReadStdoutTrimNewlines()
 	assert.Contains(wb, "10000")
 }
+
+func TestWalletExportImportRoundTrip(t *testing.T) {
+	assert := assert.New(t)
+	require := require.New(t)
+
+	d := th.NewDaemon(t).Start()
+	defer d.ShutdownSuccess()
+
+	dw := d.RunSuccess("address", "ls").ReadStdoutTrimNewlines()
```

Comment:
should probably create a few addresses before calling ls. my guess is there might be ordering issues, but maybe not.

Reply:
The ordering on import and export shouldn't matter, the problem was the way the exported structure was serialized to json didn't match what import expected.